content/en/monitors/incident_management.md

```diff
@@ -7,30 +7,66 @@ description: Create and manage incidents
 
 In the Datadog paradigm, any of the following are appropriate situations for declaring an incident:
 
-* An issue is or may be impacting customers or services.
-* You do not know whether you should call an incident. Notify other people and increase severity appropriately.
-
-<div class="alert alert-warning">
-<a href="https://app.datadoghq.com/incidents">Incident Management</a> is currently in beta and available to all customers. For more information, reach out to sales@datadoghq.com.
-</div>
+* An issue is or may be impacting customers or internal services.
+* You do not know whether you should call an incident.
 
 ## Usage
 
-Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page for a feed of all ongoing incidents. You can configure additional fields that appear for all incidents in [Incident Settings][2].
+Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page to see a feed of all ongoing and resolved incidents. You can configure additional fields that appear for all incidents in [Incident Settings][2].
 
 ### Creating an incident
 
-You can declare an incident directly from a graph. You can also go to the upper right hand side of the Incidents page and click on “New Incident”. The following dialog box appears.
+#### From a graph
+
+You can declare an incident directly from a graph by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the graph is added to the incident as a signal. 
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a graph">}}
+
+#### From the Clipboard
+
+Use the beta [Datadog Clipboard][6] to gather multiple signals like monitors, graphs, security signals, logs, or any data in Datadog and to generate an incident. To add a graph to the clipboard, copy any graph, and then select Open Clipboard. Continuing adding all of the relevant graphs and monitors to the clipboard, select them, and then click Add to New Incident. Everything on the clipboard is added to the incident as a signal.
+
+You can also add these signals to active incidents by scrolling down and selecting the incident ID.
+
+{{< img src="monitors/incidents/from-clipboard.png" alt="Add a dashboard to the clipboard">}}
+
+{{< img src="monitors/incidents/clipboard.png" alt="Create in incident from the clipboard">}}
+
+**Note**: Data on the Clipboard can also be exported to a new or existing dashboard or a notebook in addition to an incident.
+
+#### From a monitor
+
+You can declare an incident directly from a monitor by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the monitor is added into the incident as a signal.
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a monitor">}}
```

Comment:
<img width="487" alt="Screen Shot 2020-11-19 at 2 10 37 PM" src="https://user-images.githubusercontent.com/4602523/99712441-1102cf80-2a71-11eb-8210-77b8a64cf5f7.png">

These can also be added to existing incidents